### DISCUSSION TITLE
How to cancel the wave underline for inactive code?

### DISCUSSION THREAD

**Discussion by ilovelll** [lapce/lapce#1811] [Q&A]
![image](https://user-images.githubusercontent.com/5793799/207764328-e248e591-5134-4ec5-be5d-572413f989c6.png)

See the capture

**Comment by panekj** [comment]
You can't

**Comment by MinusGix** [comment]
This is due to our highlighting of that-type-of-thing as a yellow underline, so it might be good to open an issue for this.

**Reply by panekj** [reply]
https://github.com/lapce/lapce/issues/1361
